android/app/src/main/java/com/microsoft/codepush/react/CodePushTelemetryManager.java

```diff
@@ -3,18 +3,28 @@
 import android.content.Context;
 import android.content.SharedPreferences;
 
+import com.facebook.react.bridge.ReadableMap;
 import com.facebook.react.bridge.WritableMap;
 import com.facebook.react.bridge.WritableNativeMap;
 
+import org.json.JSONException;
+import org.json.JSONObject;
+
 public class CodePushTelemetryManager {
 
     private Context applicationContext;
+    private final String APP_VERSION_KEY = "appVersion";
     private final String CODE_PUSH_PREFERENCES;
     private final String DEPLOYMENT_FAILED_STATUS = "DeploymentFailed";
```

Comment:
Same for this.. Its being used in existing working production code, I don't want to bloat this PR and introduce new risks.